Explainer/process_presentation.py

```diff
@@ -0,0 +1,53 @@
+import os
+import asyncio
+
+from pptx import Presentation
+from Explainer import read_from_powerpoint, json_utils, chat_gpt_interface
+
+
+UPLOADS_FOLDER = os.getenv('UPLOADS_FOLDER_PATH')
+
+OUTPUTS_FOLDER = os.getenv('OUTPUTS_FOLDER_PATH')
+
+async def process_slide(slides):
+    UPLOADS_FOLDER = os.environ.get("UPLOADS_FOLDER_PATH")
+    tasks = await asyncio.gather (*[asyncio.create_task(chat_gpt_interface.send_query (slide))for slide in slides])
+    return tasks
+
+async def process_presentation(file_path):
```

Comment:
Use type hints and return types -
https://www.geeksforgeeks.org/duck-typing-in-python/